src/pages/Student/Main/index.tsx

```
@@ -1,17 +1,37 @@
+
+    useEffect(() => {
+    WeeklyTimetable()
+      .then(data => setSchedule(data))
+      .catch(err => console.error('스케줄 로딩 실패:', err))
+  }, []);
+
     return (
         <s.Container>
-            <MySchedule data={STUSchedule}/>
+            <MySchedule data={schedule} /> // schedule state 전달
             <TaskGuide />       {/* 수행평가 안내*/}
```

Biome (2.1.2)

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)